_data/projects/configtips.yml

```diff
@@ -0,0 +1,17 @@
+name: Config Tips
+desc: Config Tips is an open source and collaborative library of config tips and tricks. We want to create the most comprehensive directory of config tips regardless of the tool or the environment to enable developers to make the best of their config file and increase their efficiency.
+site: https://config.tips
+tags: 
+  - config
+  - yaml
+  - git
+  - github
+  - vite
+  - aws
+  - circleci
+  - nextjs
+  - nginx
+  - packagejson
+upforgrabs:
+  name: hacktoberfest
+  link: https://github.com/CircleCI-Public/Config.Tips/issues/
```

Comment:
🤔 this should link to a list of issues in the repository that are ideal for contributors. The only issue with this label is https://github.com/CircleCI-Public/Config.Tips/issues/63 which looks like a greeting.